pkg/cmd/cli/backup/create.go

```diff
@@ -186,6 +191,27 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 		}
 	}
 
+	if o.ResPoliciesConfigmap != "" {
+		cm := &corev1.ConfigMap{}
+		if err := client.Get(context.Background(), kbclient.ObjectKey{
+			Namespace: f.Namespace(),
+			Name:      o.ResPoliciesConfigmap,
+		}, cm); err != nil {
+			return errors.Wrapf(err, "failed to get resource policies %s/%s", o.ResPoliciesConfigmap, f.Namespace())
```

Comment:
Incorrect order for policy and namespace

Reply:
updated